### ISSUE TITLE
Cannot find module 'yosay'

### ISSUE THREAD

**Issue by xiorcal** [timvw/generator-scala#4]
hello,

first of all i'm very new to yeoman and nodejs in general.

I just wanted to try the scala generator as I used it for java

here is the complete error message:

```
> cd .\sample_yo
> yo scala
readline.js:925
        throw err;
        ^

Error: Cannot find module 'yosay'
at Function.Module._resolveFilename (module.js:339:15)
at Function.Module._load (module.js:290:25)
at Module.require (module.js:367:17)
at require (internal/module.js:16:19)
at Object.<anonymous> (D:\Programmes\NodeJs\node_modules\generator-scala\app\index.js:3:13)
at Module._compile (module.js:413:34)
at Object.Module._extensions..js (module.js:422:10)
at Module.load (module.js:357:32)
at Function.Module._load (module.js:314:12)
at Module.require (module.js:367:17)
```

if you need more information i'll gladly provide them

**Comment by rgladwell** [comment]
Could we get an npm release for this fix please?

**Comment by timvw** [comment]
Done (Released 0.4.0)

**Comment by rgladwell** [comment]
@timvw much appreciated
